superchain/configs/mainnet/ink.toml

```diff
@@ -57,8 +59,9 @@ max_sequencer_drift = 600
   ProxyAdmin = "0xd56045E68956FCe2576E680c95a4750cf8241f79"
   SuperchainConfig = "0x95703e0982140D16f8ebA6d158FccEde42f04a4C"
   AnchorStateRegistryProxy = "0xde744491BcF6b2DD2F32146364Ea1487D75E2509"
-  DelayedWETHProxy = "0x14773a8040Ff22e3DcBb0C83eC8e33Be7D920D38"
+  DelayedWETHProxy = "0x3Beaca17eaE5643FB1479AA5f4B1fF75cc4b9B50"
   DisputeGameFactoryProxy = "0x10d7B35078d3baabB96Dd45a9143B94be65b12CD"
   MIPS = "0x16e83cE5Ce29BF90AD9Da06D2fE6a15d5f344ce4"
-  PermissionedDisputeGame = "0xa8E6a9bF1Ba2dF76C6787EAEbE2273Ae98498059"
+  PermissionedDisputeGame = "0x0A780bE3eB21117b1bBCD74cf5D7624A3a482963"
+  FaultDisputegame = "0x6A8eFcba5642EB15D743CBB29545BdC44D5Ad8cD"
```

Comment:
I think the CI indicates this needs to move up to line 64 to preserve the stability of the generated output (keys are in alphabetical order)

Reply:
correct

Reply:
Okay so I tried this in my own branch and its still not happy https://github.com/ethereum-optimism/superchain-registry/pull/875